controllers/reviewController.js

```diff
@@ -0,0 +1,108 @@
+const jwt = require("jsonwebtoken");
+
+const { catchAsync } = require("../utils/error");
+const reviewService = require("../services/reviewService");
+const userDao = require("../models/userDao");
+
+const getReviewData = catchAsync(async (req, res) => {
+  const accessToken = req.headers.authorization;
+  const { shopId } = req.params;
+  if (!shopId) throw new Error("KEY_DOES_NOT_EXIST_ERROR");
+
+  let userId;
+
+  if (!accessToken) userId = undefined;
+  
+  if (accessToken) {
+    let decoded;
+    try {
+      decoded = await jwt.verify(accessToken, process.env.JWT_SECRET);
+    } catch {
+      const err = new Error("INVALID TOKEN");
+      err.statusCode = 401;
+      return res.status(err.statusCode).json({ message: err.message });
+    }
+
+    const user = await userDao.getUserById(decoded.id);
+    if (!user) {
+      const err = new Error("USER_DOES_NOT_MATCH");
+      err.statusCode = 401;
+      return res.status(err.statusCode).json({ message: err.message });
+    }
+
+    req.user = user;
+    userId = req.user.userId;
+  }
+
+  const reviewData = await reviewService.getReviewData(userId, shopId);
+
+  return res.status(201).json({ reviewData });
+});
+
+const createReviewData = catchAsync(async (req, res) => {
+  const userId = req.user.userId;
+  const { content, score, shopId } = req.body;
```

Comment:
```suggestion
  const { userId } = req.user;
  const { content, score, shopId } = req.body;
```
통일성있게 똑같이 destructuring 하여 작성해주세요. 
`createReviewData` method 외에 모든 코드에 동일하게 작성되어있는데 수정 부탁드립니다~